rustls-post-quantum/Cargo.lock

```diff
@@ -441,7 +441,7 @@ dependencies = [
 
 [[package]]
 name = "rustls"
-version = "0.23.19"
+version = "0.23.20"
```

Comment:
Based on the commit message I would have expected this commit to also update to `Cargo.toml` to change:
```toml
rustls = { version = "0.23.2", features = ["aws_lc_rs"], path = "../rustls" }
```
to
```toml
rustls = { version = "0.23.20", features = ["aws_lc_rs"], path = "../rustls" }
```

Am I missing something?

Reply:
Yikes. Good catch!